Project-overview.md

```diff
@@ -0,0 +1,99 @@
+This document describes the project to make "CSS short descriptions" available to third-party tools such as code editors or developer tools.
```

Comment:
Can this file name be lower case? Is that a weird hang up? I'd never really thought about it before

Reply:
No, now you mention it, it seems totally wrong.